demo/mkldnn_quant/quant_aware/PaddleCV_mkldnn_quantaware_tutorial.md

```diff
@@ -0,0 +1,217 @@
+
+## 1. Build Paddle, PaddleSlim and Inference library from source code
+### 1.1 Build Paddle and Inference library
+```
+PADDLE_ROOT=/path/of/capi
+git clone https://github.com/PaddlePaddle/Paddle.git
+cd Paddle
+git checkout 2.0-beta -b 2.0-beta
+mkdir build
+cd build
+cmake -DFLUID_INFERENCE_INSTALL_DIR=$PADDLE_ROOT \
+      -DCMAKE_INSTALL_PREFIX=./tmp \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DWITH_PYTHON=ON \
+      -DWITH_MKL=ON \
+      -DWITH_MKLDNN=ON \
+      -DWITH_GPU=OFF  \
+      -DON_INFER=ON \
+      -DWITH_PROFILER=OFF \
+      ..
+ make -j$(nproc)
+ make inference_lib_dist
+```
+
+### 1.2 Build PaddleSlim from source code
+```
+git clone https://github.com/PaddlePaddle/PaddleSlim.git
+cd PaddleSlim
+python setup.py install
+```
+### 1.3 Use paddle and slim in sample code
+You can use paddle and paddleslim as follows:
+```
+import paddle
+import paddle.fluid as fluid
+import paddleslim as slim
+import numpy as np
+```
+
+## 2. Training and converting fp32 model to fp32 qat model
+Users can download pre-trained models at [Download pre-trained models](https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/image_classification/README.md)
+
+We provide a script to insert `fake_quantize`/`fake_dequantize` ops, train a few iterations and then save the float32 QAT model. Run as follows
+```
+python train_image_classification.py --model=ResNet50 --pretrained_model=$PATH_TO_ResNet50_pretrained --data=imagenet --data_dir=$PATH_TO_ILSVRC2012/ --save_float32_qat_dir=$PATH_TO_float32_qat_dir
+```
+Available options in the above command and their descriptions are as follows:
+- **model:** Model name. Default value: "ResNet50"
+- **pretrained_model:** A path to pre-trained model. Default value: None
+- **batch_size：** Number of training batch size. Default value is 128
+- **num_epochs:** Number of training epoches. Default value is 1
+- **config_file:** A path to training config file. Default value is `./config.yaml`
+If the user needs to change the quantization strategy, modify `config.yaml`. We sugggest the following configuration to obtain the best accuracy.
+
+```
+config = {
+         'weight_quantize_type': 'channel_wise_abs_max',
+         'activation_quantize_type': 'moving_average_abs_max',
+         'quantize_op_types': ['depthwise_conv2d', 'mul', 'conv2d', 'elmentwise_add', 'pool2d']
+     }
+```
+For better understanding of the strategies, please refer to [PaddleSlim quant_aware API](https://paddlepaddle.github.io/PaddleSlim/api/quantization_api/#quant_aware)
+
+**Note:**
+- To modify the program for quantizaiton, some training options need to be closed. Running `` sync_batch_norm '' and quant-training with multicards will cause error, the reason is unknown, for now please make sure to close it as below.
```

Comment:
`quantizaiton` -> `quantization`
`need to be closed` -> `have to be disabled` ?

Reply:
The PATH_TO_BINARY_DATA is defined about, it is from `python paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py --local --data_dir=$USER_DATASET_PATH --output_file=$PATH_TO_BINARY_DATA`